 - this is not a proper fix... should fix sage_getfile() instead

--- a/src/sage/misc/sageinspect.py
+++ b/src/sage/misc/sageinspect.py
@@ -187,7 +187,8 @@ def loadable_module_extension():
         True
     """
     # Return the full platform-specific extension module suffix
-    return import_machinery.EXTENSION_SUFFIXES[0]
+    # PATCHED for void linux, we use the shortest extension (.so)
+    return import_machinery.EXTENSION_SUFFIXES[-1]
 
 
 def isclassinstance(obj):
